### PR TITLE
adds Hermosillo chapter to config.yml and a static page

### PR DESCRIPTION
**Pull request by FernandaDguez**
# PyLadies Pull Request
_Please document the following. We will aim to be as timely as possible in processing your pull request, however as this is a volunteer staffed project anticipate at least a week for pull requests to start to be reviewed._ 

## Is this pull requesting adding or updating a [PyLadies chapter information](https://github.com/pyladies/pyladies#to-add-a-new-pyladies-location)? 

- [x] Yes
- [ ] No

## Have you confirmed the website builds locally? See [here](https://github.com/pyladies/pyladies#run-the-site-locally).

- [x] Yes
- [ ] No

##  Does your pull request pass the `netlify/pyladies/deploy-preview` check?

- [x] Yes
- [ ] No

# Pull Request Checklist

- [x] Fill out all sections above
- [x] Share pull request in PyLadies Slack channel #project-tech-infra

/cc @pyladies/tech-and-infra-admins

### CODE REVIEW

File: www/config.yml [hunk 1]
```diff
@@ -452,6 +452,19 @@ chapters:
...
   twitter: pyladies1
+- email: hermosillo@pyladies.com
+  image: pyladies-hermosillo.jpeg
+  location:
+    latitude: 29.09
+    longitude: -110.95
+  meetup: PyLadiesHMO
+  name: Hermosillo, M&eacute;xico
+  organizer: Pyladies Hermosillo
+  website: hermosillo
```
Comment (lorenanicole) [inline www/config.yml:463]: This is incorrect, please read comment in the pull request for how to remedy this.

**Review by lorenanicole — CHANGES_REQUESTED** [review]
@FernandaDguez a few things:

1. Please remove the index.html page, we no longer are hosting PyLadies location pages on pyladies.com. Instead you must open a pull request to create your official PyLadies website here: https://github.com/pyladies/chapter-websites/issues before we will add to the official website location. There we can setup your custom domain, which will be the link you use in the www/config.yml file under website
2. Register your PyLadies chapter in the Chapter Directory here. You must be logged in with your PyLadies google account to access the form.
3. Once the above is complete, we will review this pull request again.

File: www/locations/hermosillo/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{% extends "site.html" %}
```
Comment (lorenanicole) [inline www/locations/hermosillo/index.html:1]: Remove this file completely. Instead the `website` will point to your custom domain. For an example see: https://github.com/pyladies/pyladies/blob/98e45e86635d4fb982b662d91eec7e904ab113f7/www/config.yml#L279.

**Review by lorenanicole — CHANGES_REQUESTED** [review]

**Comment by lorenanicole** [comment]
@FernandaDguez you can now add this to your file - https://github.com/pyladies/pyladies/blob/98da2348ef545ffb1b4e5c5b68e26945678f384b/www/config.yml#L397 - and link to your chapter's website - https://hermosillo.pyladies.com!

**Comment by FernandaDguez** [comment]
Hello, is there something else needed to merge this?

**Review by Mariatta — APPROVED** [review]
Thanks. Sorry for the delay.

**Pull request merged**